src/Cmts.ml

```diff
@@ -481,9 +481,9 @@ let split_asterisk_prefixed (txt, {Location.loc_start}) =
   let pat =
     String.Search_pattern.create
       (String.init len ~f:(function
-        | 0 -> '\n'
-        | n when n < len - 1 -> ' '
-        | _ -> '*' ))
```

Comment:
And here the red indentation is preferred?

Reply:
yes